test_scripts/Polices/build_options/TC003_REQUEST_PTU_Trigger_App_Excluded_PT_HTTP.lua

```diff
@@ -62,31 +66,28 @@ function Test:TestStep_StartNewSession()
 end
 
 function Test:TestStep_PTU_AppID_SecondApp_NotListed_PT()
-  local is_test_passed = true
   local correlationId = self.mobileSession1:SendRPC("RegisterAppInterface", config.application2.registerAppInterfaceParams)
 
+  self.mobileSession1:ExpectResponse(correlationId, { success = true, resultCode = "SUCCESS"})
+  self.mobileSession1:ExpectNotification("OnHMIStatus", {hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN"})
+
   EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application2.registerAppInterfaceParams.appName } })
-  :Do(function()
-
-    EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate",
-          { status = "UPDATE_NEEDED" }, {status = "UPDATING"}):Times(2)
-    :Do(function(_,data)
-      if(data.params.status == "UPDATE_NEEDED") then
-        is_test_passed = testCasesForPolicyTableSnapshot:verify_PTS(true,
-                { config.application1.registerAppInterfaceParams.appID, config.application2.registerAppInterfaceParams.appID, },
-                {config.deviceMAC},
-                {""},
-                "print")
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", { status = "UPDATE_NEEDED" }, {status = "UPDATING"}):Times(2)
+
+  self.mobileSession1:ExpectNotification("OnSystemRequest")
+  :Do(function(_, data)
+      print("SDL->MOB1: OnSystemRequest()", data.payload.requestType)
+      if data.payload.requestType == "HTTP" then
+        request_http_received = true
+        ptu = json.decode(data.binaryData)
       end
     end)
-    EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "HTTP"})
+  :Times(2)
+end
 
-    if(is_test_passed == false) then
-      self:FailTestCase("Test is FAILED. See prints.")
-    end
-  end)
-  self.mobileSession1:ExpectResponse(correlationId, { success = true, resultCode = "SUCCESS"})
-  self.mobileSession1:ExpectNotification("OnHMIStatus", {hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN"})
+function Test:ValidatePTS()
+  if(request_http_received == false) then self:FailTestCase("OnSystemRequest , requestType: HTTP is not received at all") end
+  if(ptu == nil) then self:FailTestCase("Binary data is empty") end
```

Comment:
@istoimenova, Seems I was wrong and check on blank value should be done not for ptu, but for data.binaryData
Because now if binaryData is blank json.decode() function fails with panic error